<ui version="4.0" >
 <class>MinimizeEnergyPropDialog</class>
 <widget class="QDialog" name="MinimizeEnergyPropDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>3</hsizetype>
    <vsizetype>3</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>300</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Minimize Energy</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="5" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>221</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QToolButton" name="whatsthis_btn" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>41</width>
         <height>23</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn" >
       <property name="toolTip" >
        <string>Cancel</string>
       </property>
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok_btn" >
       <property name="toolTip" >
        <string>OK</string>
       </property>
       <property name="text" >
        <string>Minimize Energy</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="buttonGroup8_2" >
     <property name="title" >
      <string>Minimize physics engine</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <property name="spacing" >
       <number>4</number>
      </property>
      <item>
       <widget class="QComboBox" name="minimize_engine_combobox" >
        <property name="toolTip" >
         <string>Choose the simulation engine with which to minimize energy.</string>
        </property>
        <item>
         <property name="text" >
          <string>NanoDynamics-1 (Default)</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>GROMACS with ND1 Force Field</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Background GROMACS with ND1 Force Field</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="buttonGroup8" >
     <property name="title" >
      <string>Minimize Options</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item>
       <widget class="QRadioButton" name="minimize_all_rbtn" >
        <property name="toolTip" >
         <string>Perform energy minimization on all the atoms in the workspace</string>
        </property>
        <property name="text" >
         <string>Minimize all</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="minimize_sel_rbtn" >
        <property name="toolTip" >
         <string>Perform energy minimization on only the atoms that have been selected</string>
        </property>
        <property name="text" >
         <string>Minimize selection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="electrostaticsForDnaDuringMinimize_checkBox" >
        <property name="text" >
         <string>Electrostatics for DNA reduced model</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="enableNeighborSearching_check_box" >
        <property name="text" >
         <string>Enable neighbor searching (slow but accurate)</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="watch_motion_groupbox" >
     <property name="title" >
      <string>Watch motion in real time</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="update_asap_rbtn" >
        <property name="toolTip" >
         <string>Update every 2 seconds, or faster if it doesn't slow adjustments by more than 20%</string>
        </property>
        <property name="text" >
         <string>Update as fast as possible</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>2</number>
        </property>
        <item>
         <widget class="QRadioButton" name="update_every_rbtn" >
          <property name="toolTip" >
           <string>Specify how often to update the screen during adjustments</string>
          </property>
          <property name="text" >
           <string>Update every</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="update_number_spinbox" >
          <property name="toolTip" >
           <string>Specify how often to update the screen during adjustments</string>
          </property>
          <property name="maximum" >
           <number>9999</number>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="value" >
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="update_units_combobox" >
          <property name="toolTip" >
           <string>Specify how often to update the screen during adjustments</string>
          </property>
          <item>
           <property name="text" >
            <string>frames</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>seconds</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>minutes</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>hours</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QGroupBox" name="groupBox20" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Convergence criteria</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <property name="spacing" >
       <number>4</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="endrms_lbl" >
          <property name="toolTip" >
           <string>Target RMS force (pN)</string>
          </property>
          <property name="text" >
           <string>EndRMS:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="endmax_lbl" >
          <property name="toolTip" >
           <string>Target max force (pN)</string>
          </property>
          <property name="text" >
           <string>EndMax:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="cutoverrms_lbl" >
          <property name="toolTip" >
           <string>Cutover RMS force (pN)</string>
          </property>
          <property name="text" >
           <string>CutoverRMS:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="cutovermax_lbl" >
          <property name="toolTip" >
           <string>Cutover max force (pN)</string>
          </property>
          <property name="text" >
           <string>CutoverMax:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>2</number>
        </property>
        <item>
         <widget class="QDoubleSpinBox" name="endRmsDoubleSpinBox" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>3</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="suffix" >
           <string> pN</string>
          </property>
          <property name="decimals" >
           <number>3</number>
          </property>
          <property name="maximum" >
           <double>501.000000000000000</double>
          </property>
          <property name="value" >
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="endMaxDoubleSpinBox" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>3</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="suffix" >
           <string> pN</string>
          </property>
          <property name="decimals" >
           <number>2</number>
          </property>
          <property name="maximum" >
           <double>2501.000000000000000</double>
          </property>
          <property name="value" >
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="cutoverRmsDoubleSpinBox" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>3</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="suffix" >
           <string> pN</string>
          </property>
          <property name="decimals" >
           <number>2</number>
          </property>
          <property name="maximum" >
           <double>12500.000000000000000</double>
          </property>
          <property name="value" >
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="cutoverMaxDoubleSpinBox" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>3</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="suffix" >
           <string> pN</string>
          </property>
          <property name="decimals" >
           <number>2</number>
          </property>
          <property name="maximum" >
           <double>60001.000000000000000</double>
          </property>
          <property name="value" >
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="restore_btn" >
       <property name="text" >
        <string>Restore Defaults</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
